app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -163,6 +164,10 @@ fun MapScreen(
   // Mutable state to store the PointAnnotationManager for parking labels
   var pLabelAnnotationManager by remember { mutableStateOf<PointAnnotationManager?>(null) }
 
+  var showPreviewCard by remember { mutableStateOf(false) }
```

Comment:
It is a non blocking issue but since all other variables have documentation maybe you can add one here ?

Reply:
Sure, i'll do it for consistency. But I believe that sometimes, trivial comments should be avoided. The name of the variable is self explanatory.